zephyr/wrapper.c

```diff
@@ -624,7 +625,10 @@ int task_main_start(struct sof *sof)
 	 * (only called from single core, no RMW lock)
 	 */
 	__ASSERT_NO_MSG(cpu_get_id() == PLATFORM_PRIMARY_CORE_ID);
-
+#if defined(CONFIG_PM)
```

Comment:
Better to stay with one version of ifdef
#if defined(CONFIG_PM)
#ifdef CONFIG_PM_POLICY_CUSTOM